lib/balena-lint.ts

```diff
@@ -254,8 +254,8 @@ export const lint = async (passedParams: any) => {
 	if (options.argv.u) {
 		const depcheck = await import('depcheck');
 		await Promise.all(
-			options.argv._.map(async (dir: string) => {
-				dir = await getPackageJsonDir(dir);
+			options.argv._.map(async (dir) => {
+				dir = await getPackageJsonDir(`${dir}`);
```

Comment:
This change was caught by strict mode, since the argument will actually be parsed to a number if it looks like one - in practice I doubt we target any dirs that look like numbers but this will make it safer if someone decides to put their code in a `1337` folder